plugins/org.csstudio.opibuilder.widgets.edm/src/org/csstudio/opibuilder/widgets/edm/figures/EdmSymbolFigure.java

```diff
@@ -40,9 +40,14 @@ protected void paintClientArea(Graphics graphics) {
         if (image != null) {
             // If what were trying to draw is out of bounds, simply draw the full image to
             // fill the area
-            if ((1 + subImageSelection) * subImageWidth > image.getBounds().width
-                    || subImageSelection * subImageWidth < 0 || getClientArea().x < 0 || getClientArea().y < 0
+            if (subImageSelection * subImageWidth < 0 || getClientArea().x < 0 || getClientArea().y < 0
                     || getClientArea().width < 0 || getClientArea().height < 0) {
+                graphics.fillRectangle(getClientArea().x, getClientArea().y, getClientArea().width,
+                        getClientArea().height);
+            }
+            // If the subImage width is greater than the image width, just show entire
```

Comment:
This makes sense. It might be worth logging something as this is clearly a misconfiguration. But I'm not sure whether we do log things for general opi problems like that, it could be quite noisy.